CrewChiefV4/ui_text.txt

```diff
@@ -593,6 +609,11 @@ pcars64_launch_exe_help = This is the location of Steam.exe or the direct path t
 pcars64_launch_exe_filter = PCARS_64BIT
 pcars64_launch_exe_category = UI_STARTUP_AND_PATHS
 
+f1_2018_launch_exe = F1 2018 launch exe
+f1_2018_launch_exe_help = This is the location of Steam.exe or the direct path to f1_2018.exe. Use forward slashes to separate paths.
+f1_2018_launch_exe_filter = F1_2018
```

Comment:
Probably not worth doing now, but, something to keep in mind.

Currently, bunch of properties that don't work in F1 2018 will show up if game is selected.  The fix is to set filter of those to _filter = !F1_2018 (or, !F1_2018;IRACING for those that already excluded from IRACING)  Note to self: need to check if _filter = !IRACING still treats property as common for other games.